ReactWindows/ReactNative.Tests/ReactInstanceManagerTests.cs

```diff
@@ -564,6 +567,8 @@ private static ReactInstanceManager CreateReactInstanceManager()
 
         private static ReactInstanceManager CreateReactInstanceManager(string jsBundleFile, LifecycleState initialLifecycleState = LifecycleState.Foreground)
         {
+            ReactNative.Bridge.DispatcherHelpers.Initialize();
```

Comment:
please add ```using ReactNative.Bridge``` to the top of the file and ditch the full qualified namespace. if you did this because it conflicts with the Test version of DispatcherHelpers, please rename DispatcherHelpers as part of this PR.

Reply:
The UWP version of test DispatcherHelpers is referenced in ~140 places, and the WPF one in 10 places.
I'm not against renaming, especially when names clash and that can confuse readers (like me), but I'm not sure why it has to be done as part of this particular PR.

I could fix in a separate PR (though I need some agreement from @rozele, etc., I won't like to go in a dead end).

---
In reply to: [169538127](https://github.com/Microsoft/react-native-windows/pull/1653#discussion_r169538127) [](ancestors = 169538127)